subsys/bluetooth/audio/media_proxy.c

```diff
@@ -197,7 +197,7 @@ uint8_t media_proxy_sctrl_get_media_state(void)
 	return mprx.local_player.calls->get_media_state();
 }
 
-void media_proxy_sctrl_send_command(struct mpl_cmd cmd)
+void media_proxy_sctrl_send_command(struct mpl_cmd *cmd)
```

Comment:
Could `cmd` be `const` here as well? Ideally the command handler should not need to to modify the command.